layouts/default.vue

```diff
@@ -85,7 +85,8 @@ export default {
       { title: 'Login', icon: 'mdi-login', ref: '/login', auth: false, menu: true },
       { title: 'Logout', icon: 'mdi-logout', ref: null, logout: true, auth: true, menu: true },
       { title: 'Editar Regras', icon: 'assignment', ref: '/rules/edit', auth: true, menu: false },
-      { title: 'Regras', icon: 'chrome_reader_mode', ref: '/rules', auth: null, menu: false }
+      { title: 'Regras', icon: 'chrome_reader_mode', ref: '/rules', auth: null, menu: false },
+      { title: 'Meses de Disputa', icon: 'mdi-calendar', ref: '/disputes', auth: true, menu: false }
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rafaelmbcosta/league-nuxt/pullRequest?prid=7487498)
```suggestion
      { title: "Meses de Disputa", icon: 'mdi-calendar', ref: '/disputes', auth: true, menu: false }
```